### DISCUSSION TITLE
Question about 'Number of downloads'?

### DISCUSSION THREAD

**Discussion by Someonemustnothavethis** [TumblThreeApp/TumblThree#337] [Q&A (Question & Answer)]
What's the difference between 'Downloaded Files' and 'Number of Downloads'?  I've read everything but there's no explanation of what it is.
Thanks!
By the way, it's been running perfectly for months with no problems.  I use it once a month to catch up with ~220 blogs.  It sure beats going through each one individually.

**Comment by thomas694** [comment]
"Number of Downloads" contains the total number of possible downloads according to the selected settings.
"Downloaded Files" contains how many of these already have been downloaded.

**Comment by Someonemustnothavethis** [comment]
Thanks for the answer.  It brings more questions to mind though...I hope you don't mind (*chuckle*).
I assume that both are cumulative?  I see that in many blogs, "# of Downloads" is quite small compared to the number actually downloaded.  Others show similar numbers with "# of downloads" being a bit less than "Downloaded Files", as I would expect.
I'd like to suggest that the name be changed to something like "Potential" or "Possible Downloads" which might be more self explanatory.

My copy is still grinding along downloading; it's grabbed (so far) just over 200 GB and a million images.  I added a bunch of new ones this month and deleted a bunch that had been abandoned.

**Comment by thomas694** [comment]
Well, they are kind of cumulative. The best numbers are shown after adding the blog or the first run.
Over time, especially if settings are changed or doing plenty runs, they can get a bit wrong...
I wouldn't take the numbers too serious, there's still room for one or the other improvement.

**Comment by Someonemustnothavethis** [comment]
Thanks again,
Perhaps it might be best to eliminate the column '# of Downloads".  "Downloaded Files" could be reset individually or collectively
if a person desired.  As I only download once a month, it would be handy for me to see how many new images I'd grabbed that month. 
Thanks again for your work on this and for taking the time to answer my questions.
Rob.

**Comment by thomas694** [comment]
There's now a new column "newly downloaded items since last run".

**Comment by thomas694** [comment]
The issue has been fixed and closed. You can still comment. Feel free to ask for reopening the issue if needed.
